### ISSUE TITLE
Stopping container outputs JS syntax error

### ISSUE THREAD

**Issue by nickcmaynard** [Inrixia/Floatplane-Downloader#198]
```
Apr 29 06:13:56 chibi systemd[1314]: Stopping container-floatplane-downloader.service - Podman container-floatplane-downloader.service...
Apr 29 06:13:56 chibi floatplane-downloader[440655]: node:internal/errors:541
Apr 29 06:13:56 chibi floatplane-downloader[440655]:       throw error;
Apr 29 06:13:56 chibi floatplane-downloader[440655]:       ^
Apr 29 06:13:56 chibi floatplane-downloader[440655]: 
Apr 29 06:13:56 chibi floatplane-downloader[440655]: TypeError [ERR_INVALID_ARG_TYPE]: The "code" argument must be of type number. Received type string ('SIGTERM')
Apr 29 06:13:56 chibi floatplane-downloader[440655]:     at process.set [as exitCode] (node:internal/bootstrap/node:123:9)
Apr 29 06:13:56 chibi floatplane-downloader[440655]:     at process.exit (node:internal/process/per_thread:182:24)
Apr 29 06:13:56 chibi floatplane-downloader[440655]:     at process.emit (node:events:519:28) {
Apr 29 06:13:56 chibi floatplane-downloader[440655]:   code: 'ERR_INVALID_ARG_TYPE'
Apr 29 06:13:56 chibi floatplane-downloader[440655]: }
Apr 29 06:13:56 chibi floatplane-downloader[440655]: 
Apr 29 06:13:56 chibi floatplane-downloader[440655]: Node.js v21.7.2
Apr 29 06:13:56 chibi podman[1042001]: 2024-04-29 06:13:56.762269364 +0100 BST m=+0.038642140 container died aa9139d93d24918b4ca230d77342c14a667f6bb9a7b6c590e14bbb638a8c042c (image=docker.io/inrix/floatplane-downloader:la>
Apr 29 06:13:57 chibi podman[1042001]: 2024-04-29 06:13:57.891686114 +0100 BST m=+1.168058915 container remove aa9139d93d24918b4ca230d77342c14a667f6bb9a7b6c590e14bbb638a8c042c (image=docker.io/inrix/floatplane-downloader:>
Apr 29 06:13:57 chibi podman[1041988]: aa9139d93d24918b4ca230d77342c14a667f6bb9a7b6c590e14bbb638a8c042c
Apr 29 06:13:57 chibi systemd[1314]: container-floatplane-downloader.service: Main process exited, code=exited, status=1/FAILURE
Apr 29 06:13:57 chibi systemd[1314]: container-floatplane-downloader.service: Failed with result 'exit-code'.
Apr 29 06:13:57 chibi systemd[1314]: Stopped container-floatplane-downloader.service - Podman container-floatplane-downloader.service.
Apr 29 06:13:57 chibi systemd[1314]: container-floatplane-downloader.service: Consumed 45.009s CPU time.
```

This is as a result of running `podman stop --ignore`.

I suspect this is as simple as giving process.exit() a numeric argument (perhaps `1`?)

**Comment by Inrixia** [comment]
Yep, thx will look at sorting this soon

**Comment by nickcmaynard** [comment]
Fixed by #200
